### ISSUE TITLE
Not possible to send notifications without summary

### ISSUE THREAD

**Issue by baj0k** [dunst-project/dunst#1071]
### Issue description
I am using a script to get the calendar info via notify-send. In that script I have a following line
`notify-send " " "$(cal --color=always | sed "s/..7m/<b><span color=\"red\">/;s/..27m/<\/span><\/b>/")"`

The first pair of quotation marks is redundant but necessary. If I don't specify a summary with at least a whitespace inside then dunst will treat the text as a summary and in result the markup won't be rendered.

I've tried using following rule in the dunstrc to remove the summary, but without success.
[cal-clock]
    format = "%b"

To summarize. I would like to have an option to send a notification without summary. I might be missing something but I think it's not possible to achieve this at this moment.

### Installation info

- Version: `1.8.1 (2022-03-02)`
- Install type: `package`
- Window manager / Desktop environment: `dwm`

<details>
<summary>Minimal dunstrc</summary>

```ini
[script-name]
    format = "%b"
```
</details>

**Comment by bebehei** [comment]
Dunst (and notify-send, part of libnotify) stick to the [notification spec](https://specifications.freedesktop.org/notification-spec/notification-spec-latest.html).

Albeit, there is no mention, that the summary is mandatory, only the `body` argument is allowed to get ignored.


Why don't you do this?

- make your sed command not use any HTML?!
- put  the output of the sed command into the summary.
- use the format `format = "<span color=red>%b</span>"`

All in all, this can't get fixed by dunst, but I think you've should have a path for a soltuion now.

**Comment by baj0k** [comment]
Right, this works. Thank you and sorry for the trouble
